core/src/utils/FilesystemUtils.cpp

```diff
@@ -42,7 +41,10 @@ bool ledger::core::FilesystemUtils::isExecutable(const std::string& path){
     auto extension=filePath.extension().string();
     return (extension==".exe") || (extension==".bat") || (extension==".com")
 #else
-    return (status(filePath).permissions() & fs::perms::owner_exec)!=fs::perms::none;
+    auto permissions=status(filePath).permissions();
+    return ((permissions & fs::perms::owner_exec)!=fs::perms::none)
+    ||((permissions & fs::perms::group_exec)!=fs::perms::none)
+    ||((permissions & fs::perms::others_exec)!=fs::perms::none);
```

Comment:
```suggestion
    return (permissions & (fs::perms::owner_exec | fs::perms::group_exec | fs::perms::others_exec))!=fs::perms::none;
```

Should work as well and makes it clearer that we test for any executable flag